x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/find.ts

```diff
@@ -48,42 +48,49 @@ export default function findBackfillTests({ getService }: FtrProviderContext) {
       expect(data.duration).to.eql('12h');
       expect(data.enabled).to.eql(true);
       expect(data.start).to.eql('2023-10-19T12:00:00.000Z');
-      expect(data.end).to.eql('2023-10-20T12:00:00.000Z');
+      expect(data.end).to.eql('2023-10-25T12:00:00.000Z');
```

Comment:
Increasing the duration of the scheduled backfill so it doesn't finish and get deleted before the find tests finish.